events/templates/events/events.html

```diff
@@ -1,74 +1,103 @@
-{% extends "events/base.html" %}
 {% load static %}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>{% block title %}Default Title{% endblock %}</title>
+    <!-- Link Bootstrap CSS -->
+    <link rel="stylesheet" href="{% static 'events/css/bootstrap.min.css' %}">
+    <!-- Your Custom CSS -->
+    <link rel="stylesheet" href="{% static 'events/css/styles.css' %}">
+    {% block extra_styles %}{% endblock %}
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Sedgwick+Ave+Display&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Caveat:wght@400;500;600;700&family=Sedgwick+Ave+Display&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <div class="banner">
+            <h1>Cheer↑</h1>
+        </div>
+        
+        
+        <nav class="navbar navbar-expand-lg navbar-light bg-light">
+            <!-- Brand/logo -->
+            <a class="navbar-brand" href="#">Cheer↑</a>
+            
+            <!-- Toggler/collapsible Button -->
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            
+            <!-- Navbar links -->
+            <div class="collapse navbar-collapse" id="navbarNav">
+                <ul class="navbar-nav">
+                    <li class="nav-item active">
+                        <a class="nav-link" href="#">Home</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">Find Out Events</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">My events</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">Contact</a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+    </header>
 
-{% block title %}
-Event List
-{% endblock %}
+    <main>
```

Comment:
Removing this line make the page showing "Default Title" on the browser tab right now. We might still need to give it a name
![image](https://github.com/gcivil-nyu-org/Wednesday-Fall2023-Team-3/assets/130470692/fca25c0d-42f6-41d3-83c6-f1ef7e0cf6d2)

Reply:
I can make that change